test/integration/etcd_storage_path_test.go

```diff
@@ -715,6 +727,14 @@ var etcdStorageData = map[schema.GroupVersionResource]struct {
 	gvr("scheduling.k8s.io", "v1alpha1", "priorityclasses"): {
 		stub:             `{"metadata":{"name":"pc1"},"Value":1000}`,
 		expectedEtcdPath: "kubernetes.io/priorityclasses/pc1",
+		expectedGVK:      gvkP("scheduling.k8s.io", "v1beta1", "PriorityClass"),
+	},
+	// --
+
+	// k8s.io/kubernetes/pkg/apis/scheduling/v1beta1
+	gvr("scheduling.k8s.io", "v1beta1", "priorityclasses"): {
+		stub:             `{"metadata":{"name":"pc2"},"Value":1000}`,
```

Comment:
surprised upper-case `Value` was accepted

Reply:
> surprised upper-case Value was accepted

copy/paste from upstream

Reply:
> surprised upper-case Value was accepted

I'm disinclined to drift from upstream.  Why do we duplicate this test anyway?

Reply:
> surprised upper-case Value was accepted

upper-case field is dropped, 0 value is allowed.

> Why do we duplicate this test anyway?

because we want to control our storage destiny and know if anything untoward happens in a rebase. for example, if something happened to break https://github.com/openshift/origin/issues/19949